### DISCUSSION TITLE
Github App Setup Help

### DISCUSSION THREAD

**Discussion by UlfJarl** [community/community#62288] [New to GitHub]
Hello. I am trying to install this self-hosted web app located at [https://github.com/jc9108/expanse](url) and developed by @jc9108. The third step in the setup "create environment variables file: cp ./backend/.env_example ./backend/.env_prod" does not do anything. No new environment variable file is created no matter how many time I put in the command. I am using macOs Monterey 12.6.8 and Terminal for the setup. For the requirements, I have git via Xcode and Docker Desktop installed. i have tried to contact the developer elsewhere and has yet to received a reply. I would appreciate any assistance in getting this app up and running.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by UlfJarl** [reply]
Unfortunately, there is no SUPPORT file and the developer/maintainer does not list his/her contact info or instruction anywhere. I have posted a new issue in the Issues section but I have not received a reply yet.

**Comment by UlfJarl** [comment]
I asked for assistance somewhere else and was able to resolve this.
